RuleDocumentation/AvoidUsingDeprecatedManifestFields.md

```diff
@@ -0,0 +1,27 @@
+#AvoidUsingDeprecatedManifestFields	
+**Severity Level: Warning**
+
+
+##Description
+
```

Comment:
Are there any other manifest properties that were deprecated? Also were they deprecated in PS v 3.0?